promscale/installation/recomm-guide.md

```diff
@@ -1,5 +1,19 @@
 # Resource recommendations
-This section describes the compute and disk requirements that are recommended for Promscale, based on the ingest rate and retention. 
+This section describes the compute and disk requirements that are 
+recommended for Promscale, based on the ingest rate and retention.
+
+**Disk size per day**: disk consumption per day based on the ingest rate.
+
+**Uncompressed buffer**: disk consumed by uncompressed data before compression.
+
+**Total disk size**: size of disk required to store the data based on the ingest rate and retention. 
+
+<highlight type="note">
+You can calculate the `total disk size` based on retention and ingest rates with this formula:
+
+Total disk size = (Disk size per day based on the ingest rate * Retention in days) + 
+Uncompressed buffer based on the ingest rate.
```

Comment:
+ WAL

Reply:
Otherwise it's not really total disk size :) or maybe document that WAL is not included for whatever reason